native/Avalonia.Native/src/OSX/AvnView.mm

```diff
@@ -219,19 +224,20 @@ - (bool) ignoreUserInput:(bool)trigerInputWhenDisabled
     {
         return TRUE;
     }
+    
+    WindowImpl* windowImpl = dynamic_cast<WindowImpl*>(_parent.getRaw());
```

Comment:
Maybe use `[NSView window]` +  `conformsToProtocol` check? Would be helpful later with (Avalonia content)->(Native content)->(Avalonia content) kind of embedding